eng/WpfArcadeSdk/Sdk/Sdk.props

```diff
@@ -14,7 +14,7 @@
 
   <PropertyGroup>
     <GenXmlStringTable>$(WpfArcadeSdkToolsDir)GenXmlStringTable.pl</GenXmlStringTable>
-    <LangVersion Condition="'$(LangVersion)'==''">8.0</LangVersion>
+    <LangVersion Condition="'$(LangVersion)'==''">preview</LangVersion>
```

Comment:
When you snap to a release branch I imagine that you're now going to have to set this to a true release or else your lang version is going to march forward in stuff like LTS branches.  You probably want to document that process somewhere.

Reply:
It's an unusual situation because of implicit global namespace feature: 

> error CS8652: The feature 'global using directive' is currently in Preview and *unsupported*. To use Preview features, use the 'preview' language version.

Reply:
Got it, so I imagine the language features will stabilize before this becomes a release branch.  Just making sure that "preview" doesn't make its way to a release accidentally.

Reply:
I've added a tracking issue here:  https://github.com/dotnet/wpf/issues/4913.

Reply:
I recommend sticking to a particular LangVersion here rather than preview. Alternatively, detect your branch name or use another breadcrumb like version string has words like 'preview' and use that to set this value to preview, and when your detect you're on a release/servicing branch, set it to a well-known stable value.

Reply:
A condition has been added to set langversion to preview for "rc" and "preview" but not release.